syntax = "proto3";

package protocol_msg;

message ProtocolMsg {
	int32 msg_type = 1;
	string user_id = 2;
	string password = 3;
	int32 authority = 4;
	int32 operation_mode = 5;
	int32 num_image = 6;
	int32 img_size = 7;
	bytes img_data = 8;
}